config/crd/bases/observability.perses.dev_perses.yaml

```diff
@@ -0,0 +1,159 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.1
+  creationTimestamp: null
+  name: perses.observability.perses.dev
+spec:
+  group: observability.perses.dev
+  names:
+    kind: Perses
+    listKind: PersesList
+    plural: perses
+    singular: perses
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: Perses is the Schema for the perses API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: PersesSpec defines the desired state of Perses
+            properties:
+              config:
+                description: 'TODO: import this from https://github.com/perses/perses/blob/main/internal/api/config/config.go#L51'
+                properties:
+                  database:
+                    properties:
+                      file:
```

Comment:
don't you want to manage the sql database ? Or maybe you prefer to do it in a future PR ?

Reply:
This will be handled in a future PR, I had some compiler trouble with the CR validation